index.md

```diff
@@ -18,10 +18,14 @@ The WSTG is a comprehensive guide to testing the security of web applications an
 
 ## Stable
 
-View the always-current stable version at [/stable](/www-project-web-security-testing-guide/stable).
+View the always-current stable version at [/stable](stable/).
 
 ## Latest
 
 We are currently developing release version 5.0.
 
-You can [read the latest development documents in our official GitHub repository](https://github.com/OWASP/wstg/tree/master/document) or view the bleeding-edge content at [/latest](/www-project-web-security-testing-guide/latest).
+You can [read the latest development documents in our official GitHub repository](https://github.com/OWASP/wstg/tree/master/document) or view the bleeding-edge content at [/latest](latest/).
+
+## Versioned Releases
+
+The only currently available versioned releases is [v4.1](v41/).
```

Comment:
```suggestion
The only currently available versioned release is [v4.1](v41/).
```

Though this makes it sound like there's never been another. Are we still making the older PDF downloads available on the Releases tab?

Reply:
I’ll tweak it